server/api/registrator.js

```diff
@@ -1,5 +1,14 @@
 const apiRetranslator = require('./api-retranslator');
 
+const handlerRegistrator = (url, app, handler) => {
+    if (handler.get) {
```

Comment:
What about a more generic form that would support other HTTP verbs?
Do you consider adopting REST?